diagnostic.md

```diff
@@ -58,18 +57,17 @@ What does typing `bin/rails dbconsole` in a directory that houses a rails app do
 us?
 
 ```md
-// your response here
-```
+open up the console with the data base that we work on. we could test our code there. it opens rails from the command line. ```
```

Comment:
This is a typo and should have been `bin/rails console` so no points will be deducted for this.